presentation/report.ts

```diff
@@ -101,32 +102,47 @@ function formatTime(blockTimestamp: number): string {
  * @param block the future block number
  */
 function estimateTime(current: Block, block: BigNumber): number {
-  if (block.lt(current.number)) throw new Error('end block is less than current')
+  if (block.lt(current.number)) return 0
+  // TODO: fix this, currently not working if the block is on another chain (e.g. arb1)
+  // throw new Error('end block is less than current')
   return block.sub(current.number).mul(13).add(current.timestamp).toNumber()
 }
 
+export function getExplorer(chainid: string): string {
```

Comment:
```suggestion
export function getExplorer(chainid: 1 | 42161): string {
```

could also do the following to have it open ended
```suggestion
export function getExplorer(chainid: 1 | 42161 | number): string {
```